packages/cxl-lumo-styles/scss/themes/vaadin-button.scss

```diff
@@ -21,3 +21,29 @@
 :host([theme="linkedin"]:hover)::before {
   opacity: 0.05;
 }
+
+:host([theme~="primary"]),
+:host([theme~="secondary"]) {
+  border: 2px solid #000;
+  border-radius: 10px;
+  box-shadow: 3px 4px 0 #000000;
```

Comment:
Please Use CSS variables for colors, sizes etc. where possible. I see you used it for font size and padding but colors should be vars too. This is a good reference `packages/cxl-lumo-styles/scss/color.scss`

![Screenshot 2025-05-19 at 17 25 35](https://github.com/user-attachments/assets/c7afeac1-1ea9-4f4a-8624-54df13988344)